site/content/docs/main/code-standards.md

```diff
@@ -11,7 +11,7 @@ When opening a pull request, please fill out the checklist supplied the template
 ## Adding a changelog
 
 Authors are expected to include a changelog file with their pull requests. The changelog file
-should be a new file created in the `changelogs/unreleased` folder. The file should follow the
+should be a new file created in the `changelogs/unreleased` folder. If your PR is for a released version of Velero, create the new file in the `changelogs/unreleased` folder in the release branch instead of on the main branch. The file should follow the
```

Comment:
I'm not sure I understand the intent of the change here. Since the changelogs/unreleased file is part of the same PR, then it will be merged into the same branch as the rest of the code changes in the PR are. For a PR against main, the changelogs/unreleased file will be merged into main. For a PR against a release branch (release-1.7, etc.), the changelogs file will be merged into thatbranch.

Reply:
I think i mixed up the wording here a bit. The original issue stemmed from someone updating the 1.7 changelog directly with a bug fix update for 1.7.1, instead of making a new file for their change. I updated this, is it better?

Reply:
Yes, the updated wording makes perfect sense to me.